File: OneDrive/Documentos/Store_Herencia/Store_Herencia/src/store/herencia/Employee.java
package store.herencia;


public class Employee extends Person {
    private String rol;
    
    public Employee(String name, String lastName, String address, String rol) {
        super(name, lastName, address);
        this.rol = rol;
    }

    public String showInfowWithRol() {
        return super.showInfo() + "rol: " + rol;
    }
    
}

File: PycharmProjects/Store_Herencia/py/Employee.py
from Person import Person

class Employee(Person):
    def __init__(self, name, last_name, address, rol):
        super().__init__(name, last_name, address)
        self.rol = rol

    def show_info_with_rol(self):
        return f"{super().show_info()}rol: {self.rol}"
